src/pages/Auth/Login.jsx

```diff
@@ -0,0 +1,10 @@
+import { LoginBody } from "../../component/LoginBody";
+import "./auth.css"
+export const Login = ()=>{
+  return(  <>
+
+       
+        <LoginBody/>
+      
+    </>)
```

Comment:
suggestion: unless you need this file for additional jsx, better to use `<LoginBody />` directly after renaming it to `<Login />`